src/main/java/com/pokegoapi/api/pokemon/HatchedEgg.java

```diff
@@ -0,0 +1,14 @@
+package com.pokegoapi.api.pokemon;
```

Comment:
for example here: https://github.com/Grover-c13/PokeGOAPI-Java/blob/Development/src/main/java/com/pokegoapi/auth/GoogleAuthJson.java